lib/saml2.coffee

```diff
@@ -39,7 +39,7 @@ create_authn_request = (issuer, assert_endpoint, destination, force_authn, conte
       '@ForceAuthn': force_authn
       'saml:Issuer': issuer
       NameIDPolicy:
-        '@Format': 'urn:oasis:names:tc:SAML:1.1:nameid-format:unspecified'
+        '@Format': nameid_format || 'urn:oasis:names:tc:SAML:1.1:nameid-format:unspecified'
```

Comment:
Generally we prefer `or` over `||` (https://github.com/Clever/coffeescript-style-guide#miscellaneous). Because this is a very minor change, I'm going to merge and change it in a follow up commit.